components/mailbox/src/Mailbox.svelte

```diff
@@ -317,8 +317,9 @@
       await dispatchDraft(event);
       return;
     }
-    dispatchEvent("threadClicked", { event, thread });
+
     currentlySelectedThread = thread;
+    thread.expanded = !thread.expanded;
```

Comment:
expand the thread after the event is dispatched from email component. This way it doesn't open in the mailbox component and again when the thread loads by itself in an email component